src/components/App.vue

```diff
@@ -123,10 +123,6 @@ export default {
         window.addEventListener('focus', event => this.onFocus(event), false);
         window.addEventListener('blur', event => this.onBlur(event), false);
         window.addEventListener('touchstart', event => this.onTouchStart(event));
-
-        if (this.sidebarState.canPin && this.$state.setting('sidebarPinned')) {
```

Comment:
Is removing the ability to pin the sidebar from user settings/`config.json` intentional?

After a while of looking, I can't find anywhere else that [`sidebarPinned`](https://github.com/kiwiirc/kiwiirc/wiki/Configuration-Options#sidebarpinned ) is checked.  Pardon if I'm misunderstanding; this is my first time looking through.